src/data/multiset/fintype.lean

```diff
@@ -0,0 +1,203 @@
+/-
+Copyright (c) 2022 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller
+-/
+import data.fintype.card
+import algebra.big_operators
+
+/-!
+# Multiset coercion to type
+
+This module defines a `has_coe_to_sort` instance for multisets and gives it a `fintype` instance.
+It also defines `multiset.to_enum_finset`, which is another way to enumerate the elements of
+a multiset. These coercions and definitions make it easier to sum over multisets using existing
+`finset` theory.
+
+## Tags
+
+multiset enumeration
+-/
+
+open_locale big_operators
+
```

Comment:
You should put a variables block with `{α : Type*} [decidable_eq α]`, since you seem to use these hypotheses in all theorems below.